tools/shell/linenoise.cpp

```diff
@@ -2980,8 +3513,8 @@ static char* linenoiseNoTTY(void) {
             }
         }
         int c = fgetc(stdin);
-        if (c == EOF || c == '\n') {
-            if (c == EOF && len == 0) {
+        if (c == -1 || c == '\n') {
```

Comment:
Why changing EOF to -1? Is this change related to multiline mode?

Reply:
There were conflicting definitions between the libraries, so I had to change it to -1. I'll make it a variable, so it is a little clearer.